sim-lib/src/random_activity.rs

```diff
@@ -48,3 +50,82 @@ impl Display for NetworkGraphView {
         write!(f, "network graph view with: {} channels", self.nodes.len())
     }
 }
+
+pub struct PaymentActivityGenerator {
+    multiplier: f64,
+    expected_payment_amt: u64,
+    source_capacity: u64,
+    event_dist: Exp<f64>,
+}
+
+impl PaymentActivityGenerator {
+    pub fn new(
+        source_capacity_msat: u64,
+        expected_payment_amt: u64,
+        multiplier: f64,
+    ) -> Result<Self, SimulationError> {
+        let lamda = events_per_month(source_capacity_msat, multiplier, expected_payment_amt)
+            / (30.0 * 24.0 * 60.0 * 60.0);
```

Comment:
sgtm, out of curiosity why a `u64` constant and then cast to `f64` rather than just `f64`? 8 bytes either way...

Reply:
Just for consistency with its meaning I guess, the amount of seconds in a month is an integer, the only reason why we cast it to float is because lamda needs to be float.